SpatialGDK/Source/SpatialGDKEditorToolbar/Private/SpatialGDKEditorToolbar.cpp

```diff
@@ -547,6 +548,18 @@ void FSpatialGDKEditorToolbarModule::VerifyAndStartDeployment()
 		return;
 	}
 
+	if (!IsSnapshotGenerated())
```

Comment:
Bit of a driveby - but seems reasonable that we should be using the GetSpatialOSSnapshotFile() both here, and when we're actually launching the deployment. Either that, or add a `snapshot to save` and `snapshot to load` setting to our editor settings.

Reply:
I'm behind this idea of using `GetSpatialOSSnapshotFile()`, although adding the `snapshot to load` and `snapshot to save` is a bit of scope creep. What do you think @bartoszrudzki-improbable

Reply:
As I check file existence by full path I will use GetSnapshotPath() method.

Reply:
@joshuahuburn if we want `snapshot to load` and `snapshot to save` functionality I think we should create other ticket for this.

Reply:
Only worry there is if a user actually sets that within the editor, we'd be checking against that name, and still launching with the `default.snapshot` name.

Reply:
Oh this is not intuitive at all. Why don't we use snapshot name from settings during launch?

Reply:
I hard coded path for default snapshot right now. As I said previously if we want `snapshot to load` and `snapshot to save` functionalities, I think it should be done in a different ticket as it requires some additional work with spot.exe (to force using specified snapshot file).

Reply:
After talk with @joshuahuburn I implemented `snapshot to load` and `snapshot to save` solution.